### ISSUE TITLE
Linkink failed while testing and building

### ISSUE THREAD

**Issue by ticksmas** [mlpack/ensmallen#245]
The following error ocurred when I was compiling ensmallen using Ubuntu 20:

```
[100%] Linking CXX executable ../ensmallen_tests
/usr/bin/ld: /usr/lib/x86_64-linux-gnu/libarmadillo.so: undefined reference for "H5Ovisit@HDF5_SERIAL_1.8.7"
```

**Comment by ticksmas** [comment]
I am not used to cmake. So, I couldn't find a way to properly fix it and contribute with the repo.

**Comment by ticksmas** [comment]
This was closed because it is a duplicated issue: #243
